environment

```diff
@@ -33,11 +35,17 @@ DSS_S3_BUCKET_TEST=org-humancellatlas-dss-test
 DSS_S3_BUCKET_TEST_FIXTURES=org-humancellatlas-dss-test-fixtures
 DSS_S3_BUCKET_STAGING=org-humancellatlas-dss-staging
 DSS_S3_BUCKET_PROD=org-humancellatlas-dss-prod
+DSS_S3_CHECKOUT_BUCKET=org-humancellatlas-dss-checkout-dev
+DSS_S3_CHECKOUT_BUCKET_TEST=org-humancellatlas-dss-checkout-test
+DSS_S3_CHECKOUT_BUCKET_TEST_FIXTURES=org-humancellatlas-dss-checkout-test-fixtures
+DSS_S3_CHECKOUT_BUCKET_STAGING=org-humancellatlas-dss-checkout-staging
+DSS_S3_CHECKOUT_BUCKET_PROD=org-humancellatlas-dss-checkout-prod
```

Comment:
Please go ahead and create these buckets in the HCA account to reserve these names.

Reply:
Done